README.md

```diff
@@ -8,7 +8,7 @@ Note this app uses govendor (https://github.com/kardianos/govendor) to manage it
 ```
 go get -d github.com/ONSdigital/go-launch-a-survey/
 cd $GOPATH/src/github.com/ONSdigital/go-launch-a-survey/
-go get -u github.com/golang/dep/cmd/dep
+brew install dep
 dep ensure
```

Comment:
I think the issue here is that `dep` isn't installed globally so we could have `$GOPATH/bin/dep ensure`